wxcrafter/wxc_widget.cpp

```diff
@@ -2696,7 +2696,7 @@ bool wxcWidget::IsParentAuiToolbar() const
 
 void wxcWidget::DoGenerateGetters(wxString& decl) const
 {
-    if(wxcSettings::Get().IsLicensed2()) {
+    if(wxcSettings::Get().IsLicensed2() && KeepAsClassMember()) {
```

Comment:
we can remove the `wxcSettings::Get().IsLicensed2()` its no longer use

Reply:
Fixed.